tests/basic/storage_pool_test.go

```diff
@@ -4585,12 +4585,169 @@ var _ = Describe("{ResizeClusterNoQuorum}", func() {
 
 			resizeErr := waitForPoolToBeResized(expectedSize, selPool.Uuid, isjournal)
 			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool %s on expansion using resize-disk", selPool.Uuid))
-
 		})
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
 
 	})
 	JustAfterEach(func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{PoolDelete}", func() {
+	/*1) Place pool on maintenence mode
+	2) Delete the pool
+	3) Add new pool
+	4) expand newly added pool*/
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolDelete", "Initiate pool deletion", nil, 0)
+
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "Initiate pool delete, then add a new pool and expand the pool"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("resiznoqr-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		stNodes := node.GetStorageNodes()
+		var nodeSelected node.Node
+		var nodePools []node.StoragePool
+		for _, stNode := range stNodes {
+			if len(nodePools) > 1 {
+				nodePools = stNode.StoragePools
+				nodeSelected = stNode
+				break
+			}
+		}
+
+		dash.VerifyFatal(len(nodePools) > 1, true, "Node has multiple storage pools?")
+		var poolToDelete node.StoragePool
+		for _, pl := range nodePools {
+			if pl.ID != 0 {
+				poolToDelete = pl
+				break
+			}
+		}
+		poolIDToDelete := fmt.Sprintf("%d", poolToDelete.ID)
+		poolsMap, err := Inst().V.GetPoolDrives(&nodeSelected)
+		log.FailOnError(err, "error getting pool drive from the node [%s]", nodeSelected.Name)
+		if _, ok := poolsMap[poolIDToDelete]; !ok {
+			log.FailOnError(fmt.Errorf("error idetifying pool drive"), "poolID %s not found in the node %s", poolIDToDelete, nodeSelected.Name)
+		}
+
+		poolsBfr, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+
+		stepLog = fmt.Sprintf("Delete poolID %s on node %s", poolIDToDelete, nodeSelected.Name)
+
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
```

Comment:
log.InfoD can be removed as Step will anyways print

Reply:
Step will print only in console
log.InfoD, we are using it to push to log file and aetos dashboard